pkg/gzip/gunzip_tinygo.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2024 the u-root Authors. All rights reserved
```

Comment:
note that the build tag is redundant maybe, since you named this _tinygo? Or does that not work in this case? No big deal.

Reply:
Maybe this is a problem with my local setup but when I run `golangci-lint` without the build tags it complains about the re-definition of `decompress`:

```
gzip.go:16:2: could not import github.com/u-root/u-root/pkg/gzip (-: # github.com/u-root/u-root/pkg/gzip
../../../pkg/gzip/gunzip_tinygo.go:16:6: decompress redeclared in this block
        ../../../pkg/gzip/gunzip.go:19:6: other declaration of decompress) (typecheck)
        "github.com/u-root/u-root/pkg/gzip"
        ^
```

Reply:
oh ho. does golangci-lint not understand the rules :-)